### ISSUE TITLE
`brew tap samangh/tap`  (not brew tap https://github.com/samangh/homebrew-tap)

### ISSUE THREAD

**Issue by dukechem** [samangh/gptfdisk#1]
In the readme.md it says:
```
brew tap https://github.com/samangh/homebrew-tap
brew install gptfdisk
```

But what worked for me on a old 10.6.8 box was:
```
 brew tap samangh/tap
 brew install gptfdisk`
```
May want to add a note to try `brew tap samangh/tap` if brew tap fails.

**Comment by dukechem** [comment]
Oh, and thanks for your work!

**Comment by samangh** [comment]
Good catch, thanks! Looks like homebrew changed how github repos are tapped. 

Fixed in 71d1591b1c656d83dc83a6d9f1bdbd2796d99662.
